semcore/modal/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.20.0] - 2023-12-05
+
+### Added
+
+- `ignorePortalsStacking` prop.
```

Comment:
Actually it's a fix because `ignorePortalsStacking` had already been in types while wasn't handled by runtime code

Reply:
fixed, and change version